ADQL.tex

```diff
@@ -425,6 +425,22 @@ \subsubsection{Whitespace}
 The rules on where whitespace is allowed and required are as in SQL-92;
 essentially, any \verb:<token>: may be followed by a \verb:<separator>:.
 
+\subsubsection{Comments}
+\label{sec:comments}
+
+As in SQL-92, comments are supported in ADQL. A comment can be anywhere in the
+query. It starts with a double minus sign and ends with the end of line.
+
```

Comment:
"anywhere in the query" is too strong a statement; in the SQL92 grammar, there's the separator nonterminal that we've (rightly) thrown out, so we can't refer to that.  What about "A comment is syntactically legal wherever whitespace is legal (and can then stand in for that whitespace)"?